operator/pkg/tasks/deinit/component.go

```diff
@@ -26,6 +26,7 @@ func NewRemoveComponentTask() workflow.Task {
 			newRemoveComponentWithServiceSubTask(constants.KarmadaWebhookComponent, util.KarmadaWebhookName),
 			newRemoveComponentWithServiceSubTask(constants.KarmadaAggregatedAPIServerComponent, util.KarmadaAggregatedAPIServerName),
 			newRemoveComponentWithServiceSubTask(constants.KarmadaAPIserverComponent, util.KarmadaAPIServerName),
+			newRemoveComponentWithServiceSubTask(constants.KarmadaMetricsAdapterComponent, util.KarmadaMetricsAdapterName),
```

Comment:
We should remove the `karmada-metrcis-adapter` first and then remove the other components.

Reply:
Adopt a LIFO installation and removal strategy?

Reply:
Although it have little effect on this, but we'd better keep the uninstall order.